Tools/Scripts/compare-results

```diff
@@ -504,6 +558,8 @@ def detectBenchmark(payload):
         return MotionMark1_1
     if detectMotionMark1_1_1(payload):
         return MotionMark1_1
+    if detectMotionMark1_2(payload):
+        return MotionMark1_2
     return None
 
 def biggerIsBetter(benchmarkType):
```

Comment:
Nit: This can be written way more efficiently without a giant `if` block (also `MotionMark1_1_1` is missing from the original):
```
def biggerIsBetter(benchmarkType):
    if benchmarkType in [JetStream2, JetStream3, Speedometer2, Speedometer3, MotionMark, MotionMark1_1, MotionMark1_1_1, MotionMark1_2]:
        return True
    elif benchmarkType in [PLT5, CompetitivePLT, PLUM3]:
        return False
    else:
        raise Exception('An unknown benchmark type was passed into biggerIsBetter: {}. It should not be possible to hit this.'.format(benchmarkType))
```